gateway/manager/targetcluster/cluster.go

```diff
@@ -0,0 +1,341 @@
+package targetcluster
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/go-openapi/spec"
+	"github.com/openmfp/golang-commons/logger"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/kcp"
+
+	appConfig "github.com/openmfp/kubernetes-graphql-gateway/common/config"
+	"github.com/openmfp/kubernetes-graphql-gateway/gateway/resolver"
+	"github.com/openmfp/kubernetes-graphql-gateway/gateway/schema"
+	kcputil "github.com/openmfp/kubernetes-graphql-gateway/listener/reconciler/kcp"
+)
+
+// FileData represents the data extracted from a schema file
+type FileData struct {
+	Definitions     map[string]interface{} `json:"definitions"`
```

Comment:
[nip] linter says it can be replaced by any

```suggestion
	Definitions     map[string]any `json:"definitions"`
```